lib/data/network/media/media_api.dart

```diff
@@ -34,4 +35,23 @@ class UploadFileAPI {
 
     return UploadFileResponse.fromJson(response);
   }
+
+  Future<UrlPreviewResponse> getUrlPreview({
+    required Uri uri,
+    int? ts,
```

Comment:
what is `ts`? I am a fan of explicit name :))

Reply:
My fault, I renamed it to `preferredPreviewTime`. It will be clearer and easier to understand for everyone